easybuild/easyblocks/l/lammps.py

```diff
@@ -211,6 +224,31 @@ def configure_step(self, **kwargs):
             if '-DFFT_PACK=' not in self.cfg['configopts']:
                 self.cfg.update('configopts', '-DFFT_PACK=array')
 
+        # Additional linker flags
+        lammps_linklibs = []
+
+        # Atomistica
+        if get_software_root('Atomistica'):
+            # link with libatomistica
+            lammps_linklibs += ['-latomistica']
+            # Atomistica needs the MPI Fortran bindings, which are not added
+            # by default in LAMMPS as it uses a C++ linker
+            if self.toolchain.options.get('usempi', None):
+                if self.toolchain.mpi_family() == toolchain.OPENMPI:
+                    # retrieve libraries used to compile OpenMPI
+                    out, ec = run_cmd("mpifort --showme:libs", simple=False)
+                    if ec:
+                        raise EasyBuildError("Failed to determine OpenMPI libraries: %s", out)
+                    lammps_linklibs += ["-l%s" % lib for lib in out.split()]
+                elif self.toolchain.mpi_family() == toolchain.INTELMPI:
+                    # add Fortran libraries used by Intel MPI
+                    lammps_linklibs += ['-lmpifort', '-lmpi']
+
+        # Set additional linker flags
+        if len(lammps_linklibs) > 0:
+            cmake_lammps_linklibs = '-DLAMMPS_LINK_LIBS="%s"' % ' '.join(lammps_linklibs)
```

Comment:
Good idea, but won't work for newer versions: https://github.com/lammps/lammps/commit/5828815b3ec50845be1a312056de559c05bbff1a

Reply:
This is very very sad :cry:  I decided to use `LAMMPS_LINK_LIBS` because I thought that this was a version agnostic approach that would not need a new revision on each release. If that is not the case, then it is just better to patch `CMakeLists.txt` instead of hacking the build from this easyblock.